src/Symfony/Component/JsonPath/Functions/LengthFunction.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\JsonPath\Functions;
+
+/**
+ * @author Alexandre Daubois <alex.daubois@gmail.com>
+ *
+ * @internal
+ */
+#[AsJsonPathFunction(name: 'length')]
+final class LengthFunction extends AbstractJsonPathFunction implements JsonPathComparableFunctionInterface
+{
+    use JsonPathFunctionArgumentTrait;
+
+    public function __invoke(array $args, mixed $context): int|false
+    {
+        $results = $args[0] ?? [];
+        $value = \is_array($results) ? ($results[0] ?? null) : $results;
+
+        return match (true) {
+            \is_string($value) => mb_strlen($value),
```

Comment:
this is JSON, it's always utf-8:
```suggestion
            \is_string($value) => mb_strlen($value, 'UTF-8'),
```